pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/question/FileQuestion.java

```diff
@@ -2,20 +2,37 @@
 
 import java.util.List;
 
+import com.google.gson.annotations.SerializedName;
 import org.broadinstitute.ddp.model.activity.instance.answer.FileAnswer;
 import org.broadinstitute.ddp.model.activity.instance.validation.Rule;
 import org.broadinstitute.ddp.model.activity.types.QuestionType;
 
 public final class FileQuestion extends Question<FileAnswer> {
 
+    @SerializedName("maxFileSize")
+    private long maxFileSize;
+
+    @SerializedName("mimeTypes")
+    private List<String> mimeTypes;
```

Comment:
Can we ensure that this is not `null` so we don't return null for this in API response?

Reply:
Fixed

Reply:
I don't see this addressed yet. How are we ensuring that it's not null? Also, let's be consistent and make this a `Set<String>` like in `FileQuestionDef`.

Reply:
It was added @NotNull in one of previous commits.

> Set<String> like in FileQuestionDef.

Done